Story2 see list of planets

cfbuckley14 opened the pull request.

Commit:
initial commit

cfbuckley14 committed:
adding basic html to index.php

Commit:
created class files

Commit:
created planet class and function to generate html

cfbuckley14 committed:
hardcoded html, added normalize and set up Sass

cfbuckley14 committed:
pulling from task 2 branch

Commit:
wrote tests, could not run, waiting on Trainer to resolve query

Commit:
added site h1

cfbuckley14 committed:
added title image, began styling of mobile view

cfbuckley14 committed:
added styling for mobile view

cfbuckley14 committed:
updates to styling

oritchie77 committed:
added media query so desktop view

cfbuckley14 committed:
final styling tweaks

oritchie77 committed:
added media query so desktop view

oritchie77 committed:
added media query so desktop view

oritchie77 committed:
added media query so desktop view

Commit:
added sorting so that planets increase in distance from the sun as you scroll down the page

cfbuckley14 committed:
updates to image sizing

oritchie77 committed:
created orbit svg class, method to generate html for svgs, styleing for solar system started

oritchie77 committed:
merged styleing form moblie view

Commit:
fixed tests and removed vendor folder from commit

oritchie77 committed:
completed styling for solar system, added if statments for ringed plannets svgs

Commit:
switched db back to docker version

oritchie77 committed:
Merge pull request #2 from iO-Academy/s2t2_-_create_sql_query_and_display_data_on_page

S2t2   create sql query and display data on page

oritchie77 committed:
added unite testing for generateOrbitSvg

oritchie77 committed:
added unite testing for generateOrbitSvg

oritchie77 committed:
added unite testing for generateOrbitSvg

oritchie77 committed:
Merge pull request #3 from iO-Academy/s2t3_desktop_solar_system_styling

S2t3 desktop solar system styling

oritchie77 committed:
removed planet card container

oritchie77 committed:
Merge pull request #5 from iO-Academy/s2t3_desktop_solar_system_styling

removed planet card container

Commit:
set up grid view on desktop

Commit:
adjusted heading sizing on mobile

cfbuckley14 committed:
Merge pull request #7 from iO-Academy/s2t4_-_grid_view_for_desktop

S2t4   grid view for desktop

cfbuckley14 committed:
adding images

cfbuckley14 committed:
chnages from final code review prior to trainer review

norayna commented on src/test/OrbitSvgTest.php:
```diff
@@ -0,0 +1,33 @@
+    public function testFailureGenerateOrbitSvg()
+    {
+        $planetName = 'earth';
+        $planetRadius = '1000';
+        $planetPosition = '1000';
+        $orbitRadius = '50346345';
```
how come this is a failure case? because the numbers are too big?

norayna commented on src/test/OrbitSvgTest.php:
```diff
@@ -0,0 +1,33 @@
+    public function testSuccessGenerateOrbitSvg()
+    {
+        $planetName = 'earth';
+        $planetRadius = '10';
+        $planetPosition = '100';
+        $orbitRadius = '50';
```
these numbers are in quotes for no reason

maxwell-01 replied:
file removed

norayna commented on src/test/OrbitSvgTest.php:
```diff
@@ -0,0 +1,33 @@
+    public function testFailureGenerateOrbitSvg()
+    {
+        $planetName = 'earth';
+        $planetRadius = '1000';
+        $planetPosition = '1000';
+        $orbitRadius = '50346345';
```
quotes?

maxwell-01 replied:
file removed

norayna commented on src/test/OrbitSvgTest.php:
```diff
@@ -0,0 +1,33 @@
+<?php
```
remove this file lol

maxwell-01 replied:
file removed

norayna commented on src/test/PlanetTest.php:
```diff
@@ -0,0 +1,28 @@
+{
+    public function testSuccessCreateCardHtml()
+    {
+        $Planet = new Planet();
+        $Planet->setName('Uranus');
+        $Planet->setType('Ice giant');
+        $Planet->setImageSrc('pretend-url');
+
```
variable name should be lowercase - $planet

maxwell-01 replied:
done

norayna commented on src/Planet.php:
```diff
@@ -0,0 +1,38 @@
+    {
+        return ('
+        <div class="planet-card">
+            <img src="images/' . $this->image_src . '">
```
needs alt attribute

maxwell-01 replied:
done

norayna commented on style.css.map:
```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["style.scss"],"names":[],"mappings":"AAAA;EACE;EACA;EACA;EACA;;;AAGF;EACE;;;AAGF;EACE;EACA;;;AAGF;EACE;EACA;;;AAGF;EACE;EACA;EACA;EACA;EACA;;AAEA;EACE;EACA;;AAGF;EACE;EACA;EACA;;AAGF;EACE;IACE;;EAEF;IACE;;EAEF;IACE;;;;AAKN;EACE;EACA;EACA;EACA;EACA;EACA;;AAEA;EACE;;AAGF;EACE;EACA;EACA;;;AAIJ;EACE;IACE;IACA;IACA;IACA;IACA;;;EAEF;IACE;;;EAEF;IACE;;;EAEF;IACE;; ...
```
this file should not be in the repo

maxwell-01 replied:
removed

norayna commented on style.scss:
```diff
@@ -0,0 +1,92 @@
+}
+
+.planet-card {
+  width: 90%;
```
should be padding on the sides instead of width

maxwell-01 replied:
done

norayna commented on index.php:
```diff
@@ -0,0 +1,48 @@
+    </div>
+    <img class="sun-image mobile" src="sun.png" alt="Sun"/>
+
+    <h1>Planet Plinky Plonk</h1>
```
two h1s

maxwell-01 replied:
removed

norayna commented on index.php:
```diff
@@ -0,0 +1,48 @@
+    <link href="https://fonts.googleapis.com/css2?family=Archivo+Black&display=swap" rel="stylesheet">
+</head>
+<body>
+<header>
+    <h1>Planet Plinky Plonk</h1>
+</header>
+<main>
+    <div class="title-image-container">
+        <img id="title-image" src="planet-plinky-plonk.png" alt="Planet Plinky Plonk"/>
+    </div>
```
the h1 and the image should be in the same place ideally

maxwell-01 replied:
done

norayna reviewed (changes requested):
delete images.zip

Commit:
updated in line with comments

norayna commented:
on my massive monitor it doesn't look great 😳 the title is too big and the background loops

![image](https://user-images.githubusercontent.com/25844355/159893212-289bed1e-638e-4df4-b16b-ab4421809bb7.png)

![image](https://user-images.githubusercontent.com/25844355/159893283-82cdb626-d97d-463c-8d91-77e8b05177b4.png)

norayna commented:
the planet type is too big compared to the planet name:
![image](https://user-images.githubusercontent.com/25844355/159894363-da8d4c56-269b-4759-8a72-ab40a06fd86b.png)

i would like it more like this:
![image](https://user-images.githubusercontent.com/25844355/159894910-8c9ede84-521e-4f50-a60d-f9b882feb99e.png)

Commit:
removed horizontal scroll from mobile

Commit:
background image will stretch

norayna commented:
on mobile, the planets are not making use of the space, i think the image should take up the full width
![image](https://user-images.githubusercontent.com/25844355/159896296-e3f40bfc-c426-49f2-a7ed-c13d531b43d3.png)

Commit:
made planet images full size on mobile

norayna commented:
style.css needs to be removed from the repo and installation instructions need to be added to the readme

norayna commented:
the logo image has uneven built in blank space on the left and right
![image](https://user-images.githubusercontent.com/25844355/159900274-837a3f2c-8ef8-421d-a6e7-37b2111473c4.png)

Commit:
wrote instructions to generate style.css
removed style.css
corrected header image to remove redundant space inside the image

Commit:
corrected readme and updated name

Commit:
updated header image

Commit:
updated header image

Commit:
added tests to isntructions

norayna reviewed (approved):
approvedddddd
